person.rb

```diff
@@ -22,4 +26,8 @@ def of_age?
     value = true if @age >= 18
     value
   end
+
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] This method should not be private, kindly consider moving this block of code before the `private` keyword on line `22`, this will make the method accessible publicly